.gitignore

```diff
@@ -1,9 +1,8 @@
 *.iml
 .gradle
 /local.properties
-/.idea/workspace.xml
-/.idea/libraries
 .DS_Store
 /build
 /captures
 .externalNativeBuild
+.idea/
```

Comment:
Why it was changed?

Also, please add back files under ./idea folder that were removed.

Reply:
Could you please explain why do we need .idea settings for our remote repo?

Reply:
To make sure everyone is using the same setup. If not, someone can have an error which we won't be able to reproduce from our side as our setup is different.

Reply:
Thanks for your explanation. Returned it back.